Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Commands/Commands.cs

```diff
@@ -457,11 +457,6 @@ public static Command DeleteAction(int actionMapIndex, string actionName)
                 InputActionSerializationHelpers.DeleteActionAndBindings(actionMap, actionID);
                 state.serializedObject.ApplyModifiedProperties();
                 state.m_Analytics.RegisterActionEdit();
-                if (state.selectedActionIndex >= actionIndex)
```

Comment:
Why is this removed?

Reply:
It shouldn't be, my bad

Reply:
This diff is no longer, marking resolved